### PR TITLE
Added more info about running & Windows Database

### PR DESCRIPTION
**Pull request by Bensikrac**
## **Description**
Added a few more lines of imformation to the getting started docs:
- How to run cockroachdb locally on windows
- How to start yarn workspace for individual modules

---

- [X] I have read the [Contributing Guide](https://docs.waldo.vision/en/getting-started/), and agree to follow the [Code of Conduct](https://docs.waldo.vision/legal/code-of-conduct/).

### CODE REVIEW

File: docs/src/pages/en/getting-started.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ If you feel comfortable self hosting, for our use case, you we require that you
 
 To start the services locally, simply run `docker compose up -d`. Or shut them off with `docker compose down`. (Run these in the repo after you have cloned it.)
 
+- For Windows you can download the Beta Archive from [Cockroachlabs](https://www.cockroachlabs.com/docs/stable/install-cockroachdb-windows.html) or [Direct zip Link](https://binaries.cockroachdb.com/cockroach-v22.2.7.windows-6.2-amd64.zip)
+  and run the exe with `./cockroach.exe start-single-node --insecure`
+- Then set the `DATABASE_URL` variable in `/packages/database/.env` and `/packages/web/.env` to the url displayed on start for postgresql
+- You even have a Web interface for management and debugging
+- WARNING BETA: (From their site: The CockroachDB executable for Windows is experimental and not suitable for production deployments. Windows 8 or higher is required.)
+
```
Comment (Huskydog9988) [inline docs/src/pages/en/getting-started.md:36]: Can you just make it extra clear this is not a recommended option for development? Also can you drop the direct zip link, we'd need to continuously update that if it remains.

**Review by Huskydog9988 — CHANGES_REQUESTED** [review]
Really like the actually running code part, totally forgot to direct people towards the other page.

File: docs/src/pages/en/getting-started.md (hunk 1 above)
Comment (Bensikrac) [reply docs/src/pages/en/getting-started.md]: Did it, idk if the red text is way too extra, or to be sure, that it doesn't cause issues later.

Comment (Huskydog9988) [reply docs/src/pages/en/getting-started.md]: Its a bit extra, but that's fine. If it ever becomes an issue we can always change it. The whole point is to keep people who don't really understand from wondering why it isn't working.

**Review by Huskydog9988 — APPROVED** [review]

**Pull request merged**